R/crimescript.R

```diff
@@ -0,0 +1,85 @@
+# Load necessary libraries
+library(tidyverse)
+library(readxl)
+library(devtools)
+
+# File paths
+ukcrime_file <- 
+  "C:/Users/ZaraMorgan/Downloads/ukcrimetables.xlsx"
+ukcrime_rda <- 
+  "C:/Users/ZaraMorgan/Downloads/ukcrimetables.rds"
+lookup_file <- 
+  "C:/Users/ZaraMorgan/OneDrive - British Red Cross Society/Local_Authority_District_to_CSPs_to_Police_Force_Areas_(December__2023)_Lookup_.xlsx"
+
```

Comment:
These files are being read off your laptop. This means that when someone else runs your script, these lines will error because they do not have access to the folder "C:/Users/ZaraMorgan/Downloads/". It also means that I am currently unable to run your script as I cannot read the dataset.

Please scrape and download the file into a temporary folder using code, rather than manually downloading the file into your downloads folder. Please use the functions `tempfile()` and `download.file()`. Please also add the website url source of the dataset as you did in `access-greenspace.R`.

See example here:

```
# Source: https://www.ons.gov.uk/peoplepopulationandcommunity/populationandmigration/populationestimates/datasets/estimatesofthepopulationforenglandandwales

population_url <- "https://www.ons.gov.uk/file?uri=/peoplepopulationandcommunity/populationandmigration/populationestimates/datasets/estimatesofthepopulationforenglandandwales/mid20222023localauthorityboundaires/mye22tablesew2023geogsv2.xlsx"

population_temp_file <- tempfile(fileext = ".xlsx")

download.file(population_url, population_temp_file, mode = "wb")

population_wales <- read_excel(population_temp_file, sheet = "MYE2 - Persons", range = "A8:D365") |>
  filter(str_starts(`Code`, "W0")) 
``